www/app/carInfo/carInfo.controller.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('budgetrentacar.carInfo')
+    .controller('carInfoController',['$scope', '$firebaseObject', 'ScannerService', 
+      function( $scope, $firebaseObject, ScannerService){
+
+    var vm = $scope;
+    
+    var FBREFERENCE = new Firebase("https://budget-cr.firebaseio.com/vehicles/"+ String(ScannerService.getCode()) + "/");
+    vm.carInformation = $firebaseObject(FBREFERENCE);
```

Comment:
```
var vm = $scope;
vm.something = stuff
```
is like saying
```
$scope.something = stuff
```

Check the correct way of doing this, the main idea of the `vm` is to NOT USE $scope.